charts/vela-core/templates/velaql/resourceTree.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: "v1"
+kind:       "ConfigMap"
+metadata:
+  name:      "application-resource-tree-view"
+  namespace: {{ include "systemDefinitionNamespace" . }}
+data:
+  template: |
+    import (
+        "vela/ql"
+    )
+    parameter: {
```

Comment:
missing the name、cluster and clusterNs fields.

Reply:
good catch, thanks.